yb-voyager/cmd/assessMigrationCommand.go

```diff
@@ -631,10 +684,19 @@ func fetchColumnsWithUnsupportedDataTypes() ([]utils.TableColumnsDataTypes, erro
 		allColumnsDataTypes = append(allColumnsDataTypes, columnDataTypes)
 	}
 
-	// filter columns with unsupported data types using srcdb.PostgresUnsupportedDataTypesForDbzm
-	pgUnsupportedDataTypes := srcdb.PostgresUnsupportedDataTypesForDbzm
+	var sourceUnsupportedDataTypes []string
+	switch source.DBType {
+	case POSTGRESQL:
+		sourceUnsupportedDataTypes = srcdb.PostgresUnsupportedDataTypesForDbzm
```

Comment:
This might not be entirely accurate, right? (we seem to be picking the unsupported types for dbzm...) 

I'm not 100% sure how to report this, because it's a little complicated, let's dicuss this